apps/wrapper/src/renderer/src/components/ViewDoc.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import logo from '../assets/logo(normal).png';
+  import exclamation from '../assets/exclamation.svg';
+
+  let hasDefaultProgram = true;
+  let launchFileFailed = false;
+  let fileExtension = 'docx'; //temp code
+
+  async function launchFile(): Promise<void> {
+    const response = await window.api.hasDefaultProgram();
+    if (response.code !== 200) {
+      hasDefaultProgram = false;
+      return;
+    }
+    const launchFileRes = await window.api.launchFile('abc123');
+    if (launchFileRes.code !== 200) {
+      launchFileFailed = true;
+      return;
+    }
+  }
+</script>
+
+{#if !hasDefaultProgram}
+  <div class="fixed flex justify-center items-center inset-0 w-full bg-black/50 z-10">
+    <div class="text-center py-6 px-12 w-1/3 bg-neutral rounded">
+      You do not have the default program to launch <span class="text-secondary"
+        >{fileExtension}</span>
```

Comment:
If prettier suggests to move the `>` to a new line, make the `>` on the same line as the element, and it's children in a new line.

```suggestion
      You do not have the default program to launch
      <span class="text-secondary">{fileExtension}</span>
```

Reply:
Changed as requested